### PR TITLE
Added Carthage Support

### PR DESCRIPTION
**Pull request by ganttastic**
Removed CocoaPods in favor of carthage for local dependency management. Added documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +181,7 @@ To run the example project, clone the repo, and run `pod install` from the Examp
 
 ## Installation
 
+### CocoaPods
```
Comment (tylermilner) [inline README.md]: Very minor. Let's put a blank line underneath so the spacing is consistent between the "Installation/CocoaPods" and "Installation/Carthage" sections.

Comment (ganttastic) [reply README.md]: Fixed!

**Comment by ganttastic** [comment]
@wmcginty Whoops! Forgot about swapping travis.

**Comment by ganttastic** [comment]
@wmcginty / @tylermilner FINALLY got the build to pass. The keychain is a pain... good for review.

**Pull request merged**